pyannote/audio/core/task.py

```diff
@@ -598,7 +598,16 @@ def prepare_data(self):
         annotations.clear()
 
         prepared_data["metadata-values"] = metadata_unique_values
-        prepared_data["metadata-labels"] = np.array(unique_labels, dtype=np.string_)
```

Comment:
I'd rather stick to `"metadata-labels"`, especially since a database may by chance be called `"global"`.